students/jeff_shabani/session07/html_render.py

```diff
@@ -8,21 +8,205 @@
 # This is the framework for the base class
 class Element(object):
 
-    def __init__(self, content=None):
-        self.content = str(content)
+    indent = " "*2
+
+    def __init__(self, content=None, tag=None, **attrs):
+        self.attrs = attrs
+        if tag:
+            self.tag = tag
+        else:
+            self.tag = ''
+        if content:
```

Comment:
Content is better left as a list, since there's a chance you will be adding multiple lines. Change this one to something like this:

```
        if content:
            self.content = [content]
        else:
            self.content = []
```